cmd/agent/app/options/options.go

```diff
@@ -108,5 +110,6 @@ func (o *Options) AddFlags(fs *pflag.FlagSet, allControllers []string) {
 	fs.DurationVar(&o.ResyncPeriod.Duration, "resync-period", 0, "Base frequency the informers are resynced.")
 	fs.IntVar(&o.ConcurrentClusterSyncs, "concurrent-cluster-syncs", 5, "The number of Clusters that are allowed to sync concurrently.")
 	fs.IntVar(&o.ConcurrentWorkSyncs, "concurrent-work-syncs", 5, "The number of Works that are allowed to sync concurrently.")
+	fs.BoolVar(&o.BuildInformerForCluster, "build-informer-for-cluster", true, "Enable this when need cluster informer.")
```

Comment:
I suggest change the flag to `--enable-cluster-resource-modeling`:
```suggestion
	fs.BoolVar(&o.EnableClusterResourceModeling, "enable-cluster-resource-modeling", true,
		"Enable means controller would build resource modeling for each cluster by syncing Nodes and Pods resources.\n"+
			"The resource modeling might be used by the scheduler to make scheduling decisions in scenario of dynamic replica assignment based on cluster free resources.\n"+
			"Disable if it does not fit your cases for better performance.")
```

Also, I'm thinking if we should add a field to `Cluster` API.

Reply:
change the flag to --enable-cluster-resource-modeling, done

Reply:
`add a field to Cluster API`, a field to declare synced resources?

Reply:
> add a field to Cluster API, a field to declare synced resources?

never mind. Let's finish this first.

Reply:
> change the flag to --enable-cluster-resource-modeling, done

I also proposed the flag `usage`, how do you think?
```
"Enable means controller would build resource modeling for each cluster by syncing Nodes and Pods resources.\n"+
			"The resource modeling might be used by the scheduler to make scheduling decisions in scenario of dynamic replica assignment based on cluster free resources.\n"+
			"Disable if it does not fit your cases for better performance."
```

Reply:
aggree